db/db.ts

```diff
@@ -884,3 +886,68 @@ export function getImageUsage(trx: KnexReadonlyTransaction): Promise<
         )
     )
 }
+
+// A topic is any tag that has a slug.
```

Comment:
nit: "...matching the slug of a published gdoc of any of the tracked gdoc type (...)"